src/utils/dao-errors.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
+
+class ResourceNotFoundError {
+  /**
+   * @param {string} resource the type of the resource
+   * @param {string} id the id request
+   */
+  constructor(resource, id) {
+    this.id = id;
+    this.resource = resource;
+  }
+
+  get [Symbol.toStringTag]() {
+    return 'ResourceNotFoundError';
+  }
+}
```

Comment:
this is the simplest way to make it possible to check error types with Chai's `should.be.a(...)` that I know of

Reply:
Nice, that looks like a better solution